GPX/GPXRoot.h

```diff
@@ -39,6 +39,9 @@
 /** Metadata about the file. */
 @property (strong, nonatomic) GPXMetadata *metadata;
 
+/** Keywords for the GPX file. */
```

Comment:
The manual says: 

> Keywords for indexing the GPX file with search engines. Comma separated.

Will do the parsing here then and make it nullable to differentiate between empty keywords and missing keywords.